R/getPlot.R

```diff
@@ -0,0 +1,108 @@
+#' Get a plot
+#'
+#' Creates and retrieves a PNG of a plot for an FcsFile. If in RStudio,
+#' displays the image.
```

Comment:
(Needs updating to reflect suggested changes.)

Reply:
:t-rex: